src/libzippp.cpp

```diff
@@ -109,9 +109,8 @@ namespace Helper {
     }
 
     static void callErrorHandlingCallback(zip* zipHandle, const std::string& msg, ErrorHandlerCallback* callback) {
-        int error_code_zip, error_code_system;
-        zip_error_get(zipHandle, &error_code_zip, &error_code_system);
-        callErrorHandlingCallbackFunc(msg, error_code_zip, error_code_system, callback);
+        auto error_code = zip_get_error(zipHandle);
```

Comment:
```suggestion
        zip_error_t* error_code = zip_get_error(zipHandle);
```

Reply:
Warning:
```
[1/3] Building CXX object CMakeFiles/libzippp.dir/src/libzippp.cpp.obj
E:/msys64/home/test/libzippp-master/src/libzippp.cpp: In function 'void Helper::callErrorHandlingCallback(zip*, const std::string&, void (*)(const std::string&, const std::string&, int, int))':
E:/msys64/home/test/libzippp-master/src/libzippp.cpp:113:22: warning: 'void zip_error_get(zip_t*, int*, int*)' is deprecated: use 'zip_get_error' instead [-Wdeprecated-declarations]
  113 |         zip_error_get(zipHandle, &error_code_zip, &error_code_system);
      |         ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from E:/msys64/home/Qrm/test/libzippp-master/src/libzippp.cpp:40:
E:/msys64/mingw64/include/zip.h:384:63: note: declared here
  384 | ZIP_DEPRECATED("use 'zip_get_error' instead") ZIP_EXTERN void zip_error_get(zip_t *_Nonnull, int *_Nullable, int *_Nullable);
      |                                                               ^~~~~~~~~~~~~
```

Reply:
@AstroAir Thanks for the info. Can you please just change as requested ?